itests/retry_test.go

```diff
@@ -0,0 +1,22 @@
+package itests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-jsonrpc"
+
+	"github.com/filecoin-project/lotus/lib/retry"
+)
+
+func TestRetryErrorIsInTrue(t *testing.T) {
+	errorsToRetry := []error{&jsonrpc.RPCConnectionError{}}
+	require.True(t, retry.ErrorIsIn(&jsonrpc.RPCConnectionError{}, errorsToRetry))
```

Comment:
Would be nice to also test a wrapped error - something like `require.True(t, retry.ErrorIsIn(xerrors.Errorf("wrapped: %w", &jsonrpc.RPCConnectionError{}), errorsToRetry))`

Reply:
good point